fury/tests/test_actors.py

```diff
@@ -310,26 +311,32 @@ def test_contour_from_roi():
         # window.show(r2)
 
 
-def test_contour_from_label():
+@pytest.mark.skipif(skip_osx, reason="This test does not work on macOS + "
+                                     "Travis. It works on a local machine"
+                                     " with 3 different version of VTK. There"
+                                     " is 2 problem to check: Travis macOS vs"
```

Comment:
there is -> there are 
problem -> problems

Reply:
Thanks for the feedback!